Functions/xASL_bids_BIDS2Legacy.m

```diff
@@ -284,6 +285,21 @@
 % Add the path to the dataPar.x struct that we return to the Master script
 dataPar.x.DataParPath = fullfile(pathLegacy, 'dataPar.json');
 
+%% 11. Clean up
```

Comment:
`Clean up` is a bit generic, do you mean here that the import-related logs are removed or moved?